src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -0,0 +1,148 @@
+      <Typography style={{ padding: '12px' }} variant="headline3">
+        Status in Access Requirements
+      </Typography>
+      <Table striped borderless bordered={false}>
+        <thead className="access-approval-header">
+          <tr>
+            <th className="thead-cell remove-border">AR ID</th>
+            <th className="thead-cell remove-border">
+              Access Requirement Name
+            </th>
+            <th className="thead-cell remove-border">Submitter</th>
+            <th className="thead-cell remove-border">Status</th>
+            <th className="thead-cell remove-border">
+              Modified Date
+              <SortIcon
+                role="button"
+                style={{ float: 'right' }}
+                active={sort.field === AccessApprovalSortField.MODIFIED_ON}
+                direction={
+                  sort.field === 'MODIFIED_ON'
+                    ? sort.direction === 'DESC'
+                      ? Direction.DESC
+                      : Direction.ASC
+                    : Direction.DESC
+                }
+                onClick={() => onSort(AccessApprovalSortField.MODIFIED_ON)}
+              />
+            </th>
+            <th className="thead-cell remove-border">
+              Expires
+              <SortIcon
+                role="button"
+                style={{ float: 'right' }}
+                active={sort.field === AccessApprovalSortField.EXPIRED_ON}
+                direction={
+                  sort.field === 'EXPIRED_ON'
+                    ? sort.direction === 'DESC'
+                      ? Direction.DESC
+                      : Direction.ASC
+                    : Direction.DESC
+                }
+                onClick={() => onSort(AccessApprovalSortField.EXPIRED_ON)}
+              />
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {accessApprovals.map(item => {
+            let modifiedOn = formatDate(moment(item.modifiedOn))
+            let expiredOn = formatDate(moment(item.expiredOn))
+            return (
+              <tr key={item.id}>
+                <td className="remove-border">
+                  <a
+                    href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!AccessRequirement:AR_ID=${item.accessRequirementId}`}
+                  >
+                    {item.accessRequirementId}
+                  </a>
+                </td>
+                <td className="remove-border">{item.accessRequirementName}</td>
+                <td className="remove-border">
+                  <UserCard size={SMALL_USER_CARD} ownerId={item.submitterId} />
+                </td>
+                <td className="remove-border">
+                  {upperFirst(item.state.toLocaleLowerCase())}
+                </td>
+                <td className="remove-border">{modifiedOn}</td>
+                <td
+                  className={`${
+                    new Date() > new Date(expiredOn) ? 'expired' : ''
```

Comment:
Not sure if this check still works now that expiredOn is a string

Reply:
This does work. However, I've run into another edge case where if the expired Date does not exist it shows the current date and time. Should I set that case with the value null?

Reply:
When the value is null does it mean that the expiration date is `never` or it does not exist?

Reply:
I would think that it means it never expires